e2e_playwright/st_markdown.py

```diff
@@ -36,10 +36,6 @@
 """
 )
 
-st.markdown("# Some header 1")
-st.markdown("## Some header 2")
-st.markdown("### Some header 3")
```

Comment:
I removed these because in the bottom of this file we have a couple of different header tests now, so these are not needed anymore